modules/claims_api/config/schemas/526.json

```diff
@@ -975,17 +975,33 @@
             }
```

Comment:
I'm guessing rswag needs re-generate our doc since this file was changed?
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/README.md#L5